### DISCUSSION TITLE
Why don't my colors look like the example pics?

### DISCUSSION THREAD

**Discussion by davidstaab** [reobin/typewritten#128] [Q&A] [answered]
How do I get the exact text and bg colors shown in the example screenshots?

<img width="571" alt="Screen Shot 2021-10-23 at 2 08 28 PM" src="https://user-images.githubusercontent.com/26676813/138568578-87d97bd0-9eb5-4cad-a733-8c4add097b61.png">

**Comment by reobin** [comment, marked as answer]
Hi @davidstaab !

From this point, you need 2 (maybe 3) things to have your terminal look exactly like the one shown in the README.

Have fun!

### 1. Font

The font used is [JetBrains Mono](https://www.jetbrains.com/lp/mono/). Install it on your machine, and update your terminal's configuration so that it uses that font.

### 2. Colors

The color scheme used in the demo is [Seoul256](https://github.com/junegunn/seoul256.vim), a vim color scheme. Check out the repo and you'll see that there are some integrations for various terminals. Install it for the terminal you're currently using.

### 3. iTerm2 (only on macOS)

This is optional, but to get the window style you can see in the demo, get [iTerm2](https://iterm2.com/) terminal. To be even more precise with the look, in the settings, navigate to `Appearance > General` and set the `Theme` to `Minimal`.
